packages/document/main-doc/docs/zh/plugin/introduction.mdx

```diff
@@ -17,14 +34,12 @@ Rsbuild 是 Modern.js 底层的构建工具，可通过添加 Rsbuild 插件修
 你可以在 `modern.config.ts` 中通过 `builderPlugins` 选项来注册 Rsbuild 插件，详见 [builderPlugins 构建插件](/configure/app/builder-plugins.html)。
 
 :::tip
-Modern.js 从 `MAJOR_VERSION.46.0` 起开始将底层的构建工具升级为 [Rsbuild](https://rsbuild.dev/)。
```

Comment:
Maybe we should keep this tip because many users are still using Modern.js <= 1.46.0 and will not be able to use Rsbuild plugin unless they upgrade.

Reply:
Okey~